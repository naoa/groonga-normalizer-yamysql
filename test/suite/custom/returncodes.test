register normalizers/yamysql

normalize NormalizerYaMySQL 'a\nb' WITH_TYPES|WITH_CHECKS
normalize NormalizerYaMySQL 'a\rb' WITH_TYPES|WITH_CHECKS
normalize NormalizerYaMySQL 'a\r\nb' WITH_TYPES|WITH_CHECKS

normalize NormalizerYaMySQLSymbolPartofspeechHtml 'a\nb' WITH_TYPES|WITH_CHECKS
normalize NormalizerYaMySQLSymbolPartofspeechHtml 'a\rb' WITH_TYPES|WITH_CHECKS
normalize NormalizerYaMySQLSymbolPartofspeechHtml 'a\r\nb' WITH_TYPES|WITH_CHECKS

normalize NormalizerYaMySQLSymbolPartofspeechHtml 'a\n' WITH_TYPES|WITH_CHECKS
normalize NormalizerYaMySQLSymbolPartofspeechHtml 'a\r' WITH_TYPES|WITH_CHECKS
normalize NormalizerYaMySQLSymbolPartofspeechHtml 'a\r\n' WITH_TYPES|WITH_CHECKS

normalize NormalizerYaMySQLSymbolPartofspeechHtml '\nb' WITH_TYPES|WITH_CHECKS
normalize NormalizerYaMySQLSymbolPartofspeechHtml '\rb' WITH_TYPES|WITH_CHECKS
normalize NormalizerYaMySQLSymbolPartofspeechHtml '\r\nb' WITH_TYPES|WITH_CHECKS

normalize NormalizerYaMySQLSymbolPartofspeechHtml 'a*b' WITH_TYPES|WITH_CHECKS
