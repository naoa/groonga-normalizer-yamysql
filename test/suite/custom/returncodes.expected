register normalizers/yamysql
[[0,0.0,0.0],true]
normalize NormalizerYaMySQL 'a\nb' WITH_TYPES|WITH_CHECKS
[[0,0.0,0.0],{"normalized":"AB","types":["alpha","alpha"],"checks":[1,2]}]
normalize NormalizerYaMySQL 'a\rb' WITH_TYPES|WITH_CHECKS
[[0,0.0,0.0],{"normalized":"AB","types":["alpha","alpha"],"checks":[1,2]}]
normalize NormalizerYaMySQL 'a\r\nb' WITH_TYPES|WITH_CHECKS
[[0,0.0,0.0],{"normalized":"AB","types":["alpha","alpha"],"checks":[1,3]}]
normalize NormalizerYaMySQLSymbolPartofspeechHtml 'a\nb' WITH_TYPES|WITH_CHECKS
[[0,0.0,0.0],{"normalized":"AB","types":["alpha","alpha"],"checks":[1,2]}]
normalize NormalizerYaMySQLSymbolPartofspeechHtml 'a\rb' WITH_TYPES|WITH_CHECKS
[[0,0.0,0.0],{"normalized":"AB","types":["alpha","alpha"],"checks":[1,2]}]
normalize NormalizerYaMySQLSymbolPartofspeechHtml 'a\r\nb' WITH_TYPES|WITH_CHECKS
[[0,0.0,0.0],{"normalized":"AB","types":["alpha","alpha"],"checks":[1,3]}]
normalize NormalizerYaMySQLSymbolPartofspeechHtml 'a\n' WITH_TYPES|WITH_CHECKS
[[0,0.0,0.0],{"normalized":"A","types":["alpha"],"checks":[1]}]
normalize NormalizerYaMySQLSymbolPartofspeechHtml 'a\r' WITH_TYPES|WITH_CHECKS
[[0,0.0,0.0],{"normalized":"A","types":["alpha"],"checks":[1]}]
normalize NormalizerYaMySQLSymbolPartofspeechHtml 'a\r\n' WITH_TYPES|WITH_CHECKS
[[0,0.0,0.0],{"normalized":"A","types":["alpha"],"checks":[1]}]
normalize NormalizerYaMySQLSymbolPartofspeechHtml '\nb' WITH_TYPES|WITH_CHECKS
[[0,0.0,0.0],{"normalized":"B","types":["alpha"],"checks":[2]}]
normalize NormalizerYaMySQLSymbolPartofspeechHtml '\rb' WITH_TYPES|WITH_CHECKS
[[0,0.0,0.0],{"normalized":"B","types":["alpha"],"checks":[2]}]
normalize NormalizerYaMySQLSymbolPartofspeechHtml '\r\nb' WITH_TYPES|WITH_CHECKS
[[0,0.0,0.0],{"normalized":"B","types":["alpha"],"checks":[3]}]
normalize NormalizerYaMySQLSymbolPartofspeechHtml 'a*b' WITH_TYPES|WITH_CHECKS
[
  [
    0,
    0.0,
    0.0
  ],
  {
    "normalized": "A B",
    "types": [
      "alpha",
      "unknown",
      "alpha"
    ],
    "checks": [
      1,
      1,
      1
    ]
  }
]
